src/components/pages/Wallet.tsx

```diff
@@ -187,7 +186,9 @@ const inviteSectionTextStyle: TextStyle = {
 };
 
 const buttonStyle: ViewStyle = {
-  marginVertical: 8
+  marginVertical: 8,
+  paddingVertical: 16,
+  width: 300
```

Comment:
minWidth? This will probably break depending on font size / i18n

Reply:
why does it need an explicit width? can't it just be made consistently good with proper padding? What motivated this change?

Reply:
if you want it to be full width you can probably just go with `width: "100%"` or something using `flex` sizing to grow to full width, depending on how this page's styles are structured. One-off random numbers like this is how we end up with giant styling headaches.